.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 ChainSafe Systems (ON)
+# SPDX-License-Identifier: LGPL-3.0-only
+
+FROM qmcgaw/godevcontainer:v0.12.0-debian
```

Comment:
Let's wait to fix the deps to upgrade to go 1.19, so that this can be bumped up to the latest image.